VERSION

```diff
@@ -1 +1 @@
-1.0.0-rc95+dev
+1.0.0+dev
```

Comment:
Is the `+dev` suffix intentional?

Reply:
There are two commits -- the first is the 1.0.0 release then one to switch back to development so builds of subsequent commits will show "+dev" rather than "1.0.0". This is what our release PRs usually look like.

Reply:
Looks like I am ready for the lunch break. Thanks :)